extension/intents/tabs/tabs.js

```diff
@@ -108,6 +108,15 @@ intentRunner.registerIntent({
   },
 });
 
+intentRunner.registerIntent({
+  name: "tabs.closeAll",
+  async run(context) {
+    const activeTabs = await context.activeTabs();
```

Comment:
Instead of creating a function, you could just put the `browser.tabs.query(...)` call right here.